app/src/main/scala/org/alephium/explorer/persistence/Migrations.scala

```diff
@@ -29,6 +29,29 @@ import org.alephium.explorer.persistence.queries.AppStateQueries
 
 object Migrations extends StrictLogging {
 
+  def addInputTxHashRefColumn(): DBActionWT[Int] = {
+    sqlu"""
+      ALTER TABLE inputs
+      ADD COLUMN IF NOT EXISTS output_ref_tx_hash bytea;
+    """
+  }
+
+  //Will automatically be recreated after the migration
```

Comment:
Why don't use `create IF NOT EXISTS` for the index?

Reply:
Index is created anyway here https://github.com/alephium/explorer-backend/blob/805da670f630e44f2e6550ba6bbf44aba241227f/app/src/main/scala/org/alephium/explorer/persistence/DBInitializer.scala#L100

Which lead to here https://github.com/alephium/explorer-backend/blob/805da670f630e44f2e6550ba6bbf44aba241227f/app/src/main/scala/org/alephium/explorer/persistence/schema/InputSchema.scala#L78-L81